packages/bpk-component-price/src/BpkPrice.js

```diff
@@ -54,35 +63,60 @@ const BpkPrice = (props: Props) => {
       // $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'decisions/flowfixme.md'.
       {...rest}
     >
-      {subtitle && (
-        <BpkText
-          className={getClassName('bpk-price__subtitle')}
-          textStyle={isSmall ? TEXT_STYLES.xs : TEXT_STYLES.sm}
-          tagName="span"
-        >
-          {subtitle}
-        </BpkText>
-      )}
+      <div
+        className={getClassName(
+          previousPrice && 'bpk-price__leadingText',
```

Comment:
any reason we cannot use kebab case here? 🙂

Reply:
You are right, I fully blame copilot here as it wrote this for me 😆